project_event/tests/test_project_event_task.py

```diff
@@ -168,3 +169,17 @@ def test_070_cancel_action(self):
         self.assertEqual(
             calendar_event.state,
             'cancelled')
+
+    def test_080_onchange_parent_id(self):
+        self.activity_1.onchange_parent_id()
```

Comment:
this is not the test of the onchange method, the test pass even if i comment the above line

Reply:
This test must check the onchange_parent_id of a task not of an activity

Reply:
Done